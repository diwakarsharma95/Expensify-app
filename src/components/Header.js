import React from 'react';

const Header = (props) => (
	<div className='header'>
		<div className='containter'>
			<h1 className='header__title'>{props.title}</h1>
			{props.subtitle && <h2 className='header__subtitle'>{props.subtitle}</h2>}
		</div>
	</div>
);

Header.defaultProps = {
	title: 'Indecision',
};

export default Header;
