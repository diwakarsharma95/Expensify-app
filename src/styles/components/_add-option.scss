.add-option-error {
	color: red;
	font-style: italic;
	padding: 0 $m-size;
	margin: $m-size 0 0 0;
}

.add-option {
	display: flex;
	flex-direction: column;
	padding: $m-size;
}

.add-option__input {
	color: $off-white;
	padding: $s-size;
	flex-grow: 1;
	background: $dark-blue;
	border: none;
	border-bottom: 0.3rem solid darken($dark-blue, 10%);
	// margin-right: $s-size;
	margin: 0 0 $s-size 0;
}

@media (min-width: $desktop-breakpoint) {
	.add-option {
		flex-direction: row;
	}
	.add-option__input {
		margin: 0 $s-size 0 0;
	}
}
