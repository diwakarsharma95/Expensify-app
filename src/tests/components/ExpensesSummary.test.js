import React from 'react';
import { shallow } from 'enzyme';
import { ExpensesSummary } from '../../components/ExpensesSummary';

test('should correctly render Expenses Summary with 1 expense', () => {
	const wrapper = shallow(<ExpensesSummary expenseCount={1} expensesTotal={235}></ExpensesSummary>);
	expect(wrapper).toMatchSnapshot();
});
test('should correctly render Expenses Summary with multiple expenses', () => {
	const wrapper = shallow(<ExpensesSummary expenseCount={23} expensesTotal={23589809809}></ExpensesSummary>);
	expect(wrapper).toMatchSnapshot();
});
