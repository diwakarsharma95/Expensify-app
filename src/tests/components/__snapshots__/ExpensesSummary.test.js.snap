// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should correctly render Expenses Summary with 1 expense 1`] = `ShallowWrapper {}`;

exports[`should correctly render Expenses Summary with multiple expenses 1`] = `ShallowWrapper {}`;
